src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
 }
+
+main {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: 708px;
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@for $daysNumber from 1 through 31 {
+  .days:nth-child(#{$daysNumber})::before {
+    content: "#{$daysNumber}";
+  }
+}
+
+@for $month-length from 28 through 31 {
+  .month-length--#{$month-length} {
+    .days:nth-child(n+#{$month-length+1}) {
+      display: none;
+    }
+  }
+}
+
+$start-day: (
+  mon: 1px,
+  tue: 102px,
+  wed: 203px,
+  thu: 304px,
+  fri: 405px,
+  sat: 506px,
+  sun: 607px
+);
+
+.days {
+  width: 78px;
+  height: 78px;
+  margin: 0.5px;
+  padding: 10px;
+  border: 1px solid #000;
+  background-color: #eee;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #000;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  @each $day, $margin in $start-day {
+    &--#{$day} {
+      margin-left: $margin;
+    }
+  }
+}
+
+.days:hover {
+  cursor: pointer;
+  transform: translateY(-20px);
+  background-color: #ffc0ca;
+  transition-duration: 0.5s;
+}
```

Comment:
It should be not here. Use scss nesting.
And **important**: [STYLES] - Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.
<img width="1083" alt="Screenshot 2022-03-14 at 14 09 51" src="https://user-images.githubusercontent.com/94289038/158191148-69951d03-cc93-415b-89c3-5efac6f8f1be.png">